www/src/components/button.js

```diff
@@ -34,6 +35,7 @@ const Button = ({
       ...(secondary && { ...buttonStyles.secondary }),
       ...(large && { ...buttonStyles.large }),
       ...(small && { ...buttonStyles.small }),
+      ...(tiny && { ...buttonStyles.tiny }),
```

Comment:
Not new here, but I'm not sure I see the value of spreading to a new object here?

```js
...(tiny && buttonStyles.tiny)
```

should do the same thing, correct?

Reply:
:+1:

Reply:
😬 👋 🙆‍♂️

Reply:
@fk tbh i have no idea what that means

Reply:
@amberleyromo I think he's referring to the fact that he originally wrote it!

@fk correct if I'm wrong, haha!

Reply:
@fk I also :)

Reply:
y'all are more emoji-native than I am!

Reply:
Yep, that was me saying "argh, hello, i did this, OK, i get it now". And putting myself in Amberley's shoes, probably wouldn't have figured it out myself. 😅